layouts/partials/comments/provider/giscus.html

```diff
@@ -1,3 +1,7 @@
+{{ $lang := .Language.LanguageCode }}
+{{ if not $lang }}
+    {{ $lang = .Language.Lang }}
+{{ end }}
```

Comment:
Hugo already does the fallback:

<img width="684" alt="image" src="https://github.com/user-attachments/assets/45a19750-4c70-422c-80b1-a0f19a930257">

https://gohugo.io/methods/page/language/

So there's no need to check this in the template. Hence the code can be simplified, and changed to:

`data-lang="{{- default $.Language.LanguageCode .lang -}}"`

Reply:
Thanks! I've updated it.